### ISSUE TITLE
Unable to load `Local task file` and `Local data file` together

### ISSUE THREAD

**Issue by patel-zeel** [microsoft/satellite-imagery-labeling-tool#28]
Hi,

As per this [guide](https://github.com/microsoft/satellite-imagery-labeling-tool/blob/main/docs/Labeler.md#to-load-a-supported-spatial-data-file), I was trying to load existing labeled `geojson` file on top of the task file on which those labels were generated but it is not working.

* Loading `Local task file` alone works.
* Loading `Local data file` alone works.
* Loading `Local data file` and then loading `Local task file` erases the existing labels.
* Loading `Local task file` first and then loading `Local data file` does not add any labels.

**Comment by rbrundritt** [comment]
Ok, so 1, 2, and 3 are all expected. Loading a task file is like opening a new session. 

#4 doesn't sound right and I can't reproduce this. Do you see any errors in the browser's dev console?

**Comment by patel-zeel** [comment]
@rbrundritt No, I could not see any error in the dev console (even after setting the log level to "All"). Let me show the screenshots from start to end.

### I load the task file
![image](https://github.com/user-attachments/assets/264e6cc7-80b7-4105-9f32-922bde32521f)

### I label a dummy bounding box and save it
![image](https://github.com/user-attachments/assets/d4af0515-532f-40d3-b00e-ab54a76663b0)

### I load the task file again and try to load the saved bounding box geojson via `Load data file` option
`Number of shapes` is still 0 as you can see and I don't see the dummy bounding box on screen as well.

![image](https://github.com/user-attachments/assets/e9082650-8e02-4b84-8211-5412b2a488e8)
